Earthfile

```diff
@@ -51,7 +51,9 @@ test-run:
     RUN ! grep 'Found tool in cache' output
     RUN cat output | grep '^::add-path::' | sed 's/::add-path:://g' > earthly-path
     RUN test "$(cat earthly-path)" = "/root/.earthly/bin"
-    RUN export PATH="$(cat earthly-path):$PATH" && earthly --version | grep '^earthly version v.*linux/amd64; Linux'
+    # [a-zA-Z0-9]* attempt to match a commit hash
+    RUN export PATH="$(cat earthly-path):$PATH" && earthly --version
+    RUN export PATH="$(cat earthly-path):$PATH" && earthly --version | grep '^earthly version v.* [a-zA-Z0-9]* linux/amd64; Alpine Linux'
```

Comment:
Running as a separate command simply makes it easier to see the version to make the necessary changes. This test relies on the specific format of our version command.